### DISCUSSION TITLE
Big UnderStrap News!

### DISCUSSION THREAD

**Discussion by howarddc** [holger1411/understrap#1272] [Announcements]
Hi all,

Thanks so much for being a part of the UnderStrap community. I'm Rob Howard, the CEO of Howard Development & Consulting, a WordPress development firm in Denver. I'm excited to announce that I've acquired the ownership/management of the UnderStrap project as of this morning.

I've posted a full announcement here: [https://rfhoward.medium.com/why-we-acquired-understrap-185df2d04178](https://rfhoward.medium.com/why-we-acquired-understrap-185df2d04178)

The tl;dr is:

- UnderStrap will always be open-source.
- We are going to invest a ton of new resources to reinvigorate the community and collaborate with all of you to get an awesome project roadmap going (including wrapping up open issues and moving toward Bootstrap 5 and considering how we want to integrate with Gutenberg).
- We'll be keeping the OverStrap themes in place while also expanding the products and services we offer on the "commercial" side as the project managers/owners, which will allow us to funnel more time and resources back into the project.

**Most importantly, I'd like to spend the month of June doing Zoom hangouts with anybody who'd like to meet and chat about the future of UnderStrap. You can book directly on my calendar right here:**

**[https://calendly.com/rfhoward/understrap](https://calendly.com/rfhoward/understrap)**

And again, I encourage you to read the full announcement, where I talk more about my philosophy and why I think we're going to really take UnderStrap to the next level: [https://rfhoward.medium.com/why-we-acquired-understrap-185df2d04178](https://rfhoward.medium.com/why-we-acquired-understrap-185df2d04178)

... also, Issues are back online again! I encourage you to speak freely, ask questions, challenge assumptions and generally do your thing in this community. We won't block or shut you down as long as your comments reasonably pertain to the future of the UnderStrap project – we want to hear all your feedback: good, bad or in between.

Looking forward to meeting you!

**Comment by UnderstrapFramework** [comment]
And this is Rob again, posting via the UnderstrapFramework username to confirm that my team and I have the keys to manage the project 👍   I'll be swapping out names/avatars/etc across all the UnderStrap accounts soon.

**Comment by Thomas-A-Reinert** [comment]
> You need to be able to work with a large, diverse team, and ensure that everyone knows they’re valued as part of the community. You need to know how to develop and ship quality software quickly and efficiently. You need to manage internal disagreements with empathy and care. And you need to be able to invest actual resources — time, money, energy and expertise — and ensure that you have a plan to make sure the project never runs out of steam. 

Well, with nothing than scorched earth that Alex left behind, I guess you can´t do any worse at least.

> Until now, UnderStrap’s monetization has come mostly from the sale of the OverStrap child themes. While we plan to keep this in place, we also see lots of potential for other products and services that complement the open-source framework. This vast potential is one of the reasons why I felt it was worth paying about three times the project’s annual earnings for the acquisition. (More on that in a moment.) From my communication with the previous project owners, I know they felt they’d taken UnderStrap as far as they could;

Well, I´m really curious about the "other products and services".
@Project owners - there has only been one to make Understrap go down the drain and that´s Alex.  In case he really stated anywhere that "_he had taken Understrap as far as he could_" - that´s probably correct. Direction: Abyss. You really can´t get any lower than that.
For potential growth? No. He commissioned some low-quality child-themes, didn´t give any support according to my knowledge and didn´t fix _any_ bug from release date on.

He didn´t even ever get any updated version released on WordPress.org with updates, bugfixes, security updates etc.pp. that the community developed. That´s not exactly what I understand by "_I have taken it as far as I could_" from my point of view. That´s basic stuff for the project, not rocket science.

> Over the next few months we’ll be surveying the community and creating a number of cool new commercial products and services related to UnderStrap, as well as releasing a separate software-as-a-service tool for WordPress developers that was already in the works prior to this acquisition.

90% Marketing blah blah that may work on medium. Any concise information you´d want to share about duration of the survey, topics/Intentions of that survey, about the AMOUNT and topic of all those "_cool new commercial products and services_" as well as about the SAAS? Too many "_smoke grenades_" (Nebelkerzen) in that line, as we say in germany. Nothing substantial, just blah blah.

> Throughout my career, I always strive to lead by example and put my time, money and energy where my mouth is. This means I don’t assume that anybody will support these commercial projects or evangelize for them out of the gate. Instead, I will endeavor to earn the community’s support by being the type of open-source project owner that I would actually want to work with as a contributor. Someone who invests heavily and unconditionally in the project, someone with the experience and empathy to lead and manage a large and diverse team, and someone with the skills to do-it-yourself when the rubber meets the road (I’ve been a full-stack developer since 1999). 

Bold words. Sound nice, really. But I´m not going to fall for bullshit-reloaded. Alex promised comparable stuff and see where we are today? Let´s judge that quote in one year from now on, okay?

**Comment by howarddc** [comment]
Thanks, @Thomas-A-Reinert! I appreciate your feedback and I don't take anything for granted – I _definitely_ hope to look back in a year and have a lot of great accomplishments to share as a group 🙂 

I don't have a ton of extra intel for you on why things went south with the previous owner/manager. I met Alex briefly during the transaction, and my understanding is that he was hired as a developer by the person who bought UnderStrap a few years ago. I did the actual financial transaction with the direct owner (which included identity verification), and Alex helped out with transferring the accounts and tech. I have seen the history (and my team and I have noticed the fact that no updates have been published to the theme despite activity on GitHub), so one of my big goals is to bring back continuity and professionalism and hopefully get us back to where things were a couple years ago, then improve from there.

I agree that the Medium post was unclear when talking about the survey. The way I look at it, there are really two groups of people I want to hear from – the first is everyone who is contributing on GitHub, and my goal is to get everybody who's able to book intro calls with me ([here is my calendar to do that](https://calendly.com/rfhoward/understrap?month=2021-06)) so we can actually spend some time together personally (at least via Zoom). I view the contributor / community work as necessarily very personal and interactive. We're also going to set up a new Slack for everyone who's contributing, assuming the group thinks that is valuable.

The second group is the much larger audience of people who've used UnderStrap or bought the commercial themes. There is quite a large e-mail list that came with the project, so those are the folks to whom I will be reaching out with the survey. That survey is basically going to ask them what products and services related to UnderStrap are most interesting to them. That's mostly just a marketing-research thing, and won't directly affect GitHub contributors.

So basically - personal time for contributors, followed by much better organization of the project, and survey for the larger "customer" audience.

Thanks again and let me know if you have any questions 👍

**Comment by howarddc** [comment]
@IanDelMar @noelspringer You now have access to the project again, as do all others who were blocked by the old admin during the sale discussion before we took ownership (I can't @ mention everyone for some reason but there were a few other people on the block list when I took over) 👍

**Comment by pattonwebz** [comment]
I hope I wasn't one of the people that were on the block list.

Also hi @howarddc, glad to have you taking over the maintainership of the project. I have seen your company around the place once or twice over the years and I feel like the project is in good hands :)

**Comment by howarddc** [comment]
Thanks, @pattonwebz!

**Comment by 0dp** [comment]
🚀

**Comment by noelspringer** [comment]
@Thomas-A-Reinert I absolutely loved your comment:

> Well, with nothing than scorched earth that Alex left behind, I guess you can´t do any worse at least.

:100: :smile:

**Comment by Thomas-A-Reinert** [comment]
Wow, a "blocklist". Just when you think it couldn´t get any lower - BAM there is Alex!
Anyway, let us not look back in anger. Sorry, if I didn´t give you a warm welcome @howarddc - but we´ve been lied to in too many places and occasions, that I can´t just jump onto the "next bandwagon" thats telling me "promising stuff". 

![image](https://user-images.githubusercontent.com/5886977/120049272-bd71cf80-c019-11eb-820d-1d1b9a526513.png)
But can I, can we? Is it not just another "business owner" telling us bullsh? Will there be any improvement? Any kind of TRUTH about what´s being told by "the owner"? How do we really know?

I´d say: @community - let´s see what the new owner comes up with. If he keeps ANY of his promises or keeps to fail like Alex did. Either he get´s anything improved or not. Either he gets a ROADMAP in place or not. Let´s see how he´s going to handle the community. Maybe he gets another deployment on WordPress.org started or not. Let´s see what happens. 
I´ll keep a low profile for the next three months at least. And will see what happens. If he´s really up to the game and loyal to the project - well, why not support it? 

Until then, I´d say @core contribs - no offense intended, but we got disappointed FAR too often! - DONT SUPPORT THAT thing AT ALL. We ALL invested so much time, heart, soul and whatever to get the project where it actually IS, that I don´t want to see anyone of you get disappointed AGAIN by some blahblah and bullshit manager! That would kinda break my heart.

@howarddc 
> We're also going to set up a new Slack for everyone who's contributing, assuming the group thinks that is valuable.

No need, there is one already. I run it and I´m willing to trade admin privs it to anyone who´s not such a BF as Alex is/was.  https://understrapworkspace.slack.com/archives/CBJJAUKR8

@howarddc Would you be able to explain why I should...
> the first is everyone who is contributing on GitHub, and my goal is to get everybody who's able to book intro calls with me (here is my calendar to do that) so we can actually spend some time together personally (at least via Zoom).

There is NO business value for any of US. That´s just YOU "offering" some kinda date where I may apply to how I want to boost your business. I´m "able" but why tf should I WANT to do that? Rethink your concept pls! YOU need to get in touch with the community not the other way around, that´s ridiculous.
Everything about that whole thing seems ridiculous! YOU need SUPPORT. WE don´t!

@howarddc  Again: Sorry in case you feel offended, I´m just speaking out loud what I think. And as you started it here - I´m going to answer it here too. In case you want a personal dialogue, I guess it´s not tooo hard to contact any of us. Feel free to contact me via my personal website.

**Comment by noelspringer** [comment]
Hi @howarddc thanks for unblocking me! I guess it was my last comment in the sale thread that was the last straw for Alex.

> and my team and I have noticed the fact that no updates have been published to the theme despite activity on GitHub

I'm glad you noticed that. I'm sure you also noticed that there are a multitude of improvements made since the 0.9.4 release on wordpress.org. 

IMO creating a new point release and publishing that to wordpress.org would be one of the first things to do.

**Comment by howarddc** [comment]
@noelspringer Thanks! We definitely have a point release at the top of our list.

@Thomas-A-Reinert I appreciate the option to use the existing Slack - can you invite me (rob@howarddc.com) when you have a moment?

Regarding the Zoom calls, there's no obligation whatsoever. I just think it's a nice way to meet and get to know everybody personally (rather than just chatting via GitHub or Slack), and I'm not looking for any particular info on those calls beyond connecting with people face to face. That said, I totally understand if it's not something you want to do, and no worries whatsoever. 

Have a great night 👍

**Comment by 0dp** [comment]
I signed up for a 20 min therapy session with Howard 😅

**Comment by 0dp** [comment]
@Thomas-A-Reinert not sure what I'm doing wrong but can't seem to get onto the slack channel 😅

**Comment by nvjacobo** [comment]
Hi,

I am understrap user from a long time. I thank all the contributors during this time.

From my perspective I would like to walk to an environment of greater transparency and trust. In this new stage I want to involve in contribute in these direction. As a sign of my appreciation for everyone who has collaborated on this project, which has uniquely benefited and enabled the projects where I use understrap. I hope @howarddc efforts has an opportunity to keep the project alive from an open and transparent perspective. 

-Jacobo

**Comment by howarddc** [comment]
Thank you, @nvjacobo!

**Comment by wildfang** [comment]
Hey there,
I'm using UnderStrap for 6 years now and I dearly, dearly hope this project isn't meant to die. 
I'm not into the topic deep enough to provide any meaningful contribution to the programming to this project, but I would be open to the idea of financing the project by selling good quality (!!!!) child themes. I could see myself buying well made child themes as a head start for commercial projects.
If I had a wish for the future of this project, it is to keep UnderStrap slim and up to date, with Bootstrap 5 etc...
Thanks so far everybody in this project for the good work. Now, please don't let the project die :-)

**Comment by howarddc** [comment]
Thanks, @wildfang! We definitely plan to keep UnderStrap active and growing for the long term 👍

**Comment by danemorgan** [comment]
Hi. I'm Dane. I'm a contractor who works with Rob and his team at Howard Development & Consulting.

I totally get where you guys are coming from and I'm not here to change minds. HDC will do that or not in the due course of time.

HDC actually made the decision, led by our lead developer ( Brian ) some weeks ago to make Understrap our official agency starter theme. When the sale came across my Twitter feed, I took notice and followed through to the listing and then to GitHub where I saw the sale thread issue. I forwarded that to Brian.

There was a brief thought of, maybe we should just create our own in-house starter theme to work with, but Rob's answer was that this was a community resource that benefitted us and that if he could preserve it, he would.

That is the kind of guy he is. Working with this team opens our eyes to different ways of thinking and looking at the tech and software world. Understrap is not a commodity, it is a resource, and it's a resource for us to protect and nurture, or to lose. That clear-cut choice is apparent. If we cut down all of the trees without planting any seedlings, we lose.

We will plant seedlings. I am excited about what comes next. I'm not going to talk out of turn on the road map, except to say, I'm looking forward to Understrap with Bootstrap 5 support.

**Comment by Thomas-A-Reinert** [comment]
Well said @danemorgan . I hope you/HDC will act acorrdingly. I've been disappointed so many times and for so long by the last "owner", that I need some proof that this leads into the right direction again. Until then, I´ll keep a low profile.

I´m not making any money with WordPress anymore though - I do governmental sites these days that need to follow WCAG 2.1 standards 100%. Standards I´d contribute to Understrap once I see some kind of recovery of this project. So for me that would be another pet project, not a thing that would save my current business or raise/deny my income.

---

@howarddc - You should have received an invitation to the slack channel/s. Please let me know if you need help. Sorry for answering so late, I had to take some "digital detox" ;)

---- 

@howarddc & @danemorgan  .. Well dunno if I´m the only one to spot any peculiarities, but it SEEMED that https://howarddc.com/ does the whole job. Now there comes a "contractor" talking about his "lead developer". And that makes the whole issue a whole lot shittier again. In my eyes at least. WE WANT TRANSPARENCY - F****G GET IT!

I understood it this way initially: https://howarddc.com/ is OWNER, **DEVELOPER**, project manager, accountant and whatsoever. Whoever may be involved in this thing _within_ this company.

And suddenly there pops up @danemorgan . Don´t get me wrong, @danemorgan I´m just speaking out loud what I´m thinking. I´m actually not trying to BLAME you. I hope you understand that.  But it kinda sucks and it´s just my personal perspective.

I don´t get to know what trade he´s coming off. I don´t get to know what company he´s coming from/representing, but he´s saying he´s a "contractor". And as it seems NOW, https://howarddc.com/ ain´t doing any development at all.  Other than it originally seemed. When I research a bit I stumble over https://danemorganmedia.com/ . And wow, that´s pure evidence of incapacity

Now another phantom named "Brian" appears. Hi Brian btw. Would you like to introduce yourself?

Again - this is getting totally confusing. Again, smoke grenades flying around. This sucks. Totally.
@howarddc  and that is exactly the reason why I don´t want to have a personal american-like chat where we both pay the utmost respect. Even if we don´t. I don´t like lies. And imma tell you where I see flaws. Love or hate it.
That´s a waste of time for me. Contact me via Mail or Slack in case you care.

**Comment by danemorgan** [comment]
@Thomas-A-Reinert, 

Here's some transparency.

I'm a contractor for HDC. HDC is a web development company. HDC has a few developers, a couple of designers, a production manager even. It's a web development company. HDC has a lead developer who provides input and guidance for the rest of the team. HDC bought the understrap/overstrap package from the previous owner.

I'm not sure how I made these things seem mutually exclusive, but they are not and I apologize for the confusion I caused you.

Until recently all of the developers at HDC were contractors. Rob offered all of us the opportunity to become employees. It is a very attractive package. I think I may be the only one who did not convert. I did not convert because I have stage 4 esophageal cancer and if I became an employee, I would have to take the company insurance plan, which is an excellent plan, but as I'm currently in the VA system, I would jeopardize some of the benefits I have through that system and thus potentially hinder my care. Rob allowed me to remain a contractor to retain those benefits.

I still take direction from Brian, the lead developer at HDC. I've been unassigned from two of HDC's clients for my primary focus of work for the company and reassigned to understrap as of this week. I'm going to be one of the people working to help the community breathe life back into this project, and I'm excited about that.

I'm uncertain how you drew the conclusion that HDC doesn't do any development. Again, the reason HDC is interested in understrap is that we recently adopted it as the main starter theme for developing new websites.

Dane Morgan Media is my own freelance domain. I registered the domain several years ago. Occasionally when I was trying to get clients there has been a website there ( most frequently built using _s ), but mostly I use it for email. Send an email to dane@ there and I'll get it. I suppose if it helps I can send you a few NameCheap invoices for that. I also own danemorgan.com and a few other domains. All of them registered through NameCheap. All of my own hosting is through Cloudways, though I don't do much of that anymore. I have a couple of other clients that I work with, though HDC is the only agency. The others are small businesses and content marketers and there aren't that many of them anymore since I have to budget my time more closely these days.

Let's see. What else. You're probably a better developer than I am, but I'll try hard and do my best. I don't give up and I'm good at researching. I love transparency in more ways than I can count. Ask a question, I'll answer it.

I'm not sure where we are on talking with the community at this point. Rob was OOTO last week, and I had a hard chemo this week, but I'm hoping to see some opportunities to start moving forward the second half of this week and early next week. As I said, I'm excited for what comes next.

My personal hopes are Bootstrap 5 and Gutenberg support, possibly with easy filters for both, but I want to hear what the community wants too. Maybe I'm way off base. I know looking through the issues that there are some dependencies that need to be looked at, and the point release thing for the .org repository seems like a critical first step.

PS I'd love an invite to the Slack dane@ that domain we discussed earlier would be brilliant. It's the address I use for most of my slacks.

**Comment by bacoords** [comment]
@Thomas-A-Reinert I'm Brian, Technology Director at HDC. I'd love an invite to the Slack as well. 

As Dane said, we've all worked together at HDC for a few years now. Some of us are full time, some are still independent contractors with our own side projects and recurring clients. All freelancers tend to leave a legacy of previous projects and small LLCs in our wakes. Rob is our founder/fearless leader at HDC, so I recommend scheduling having a call with him if you haven't already. You'll probably see more of Dane and myself as we start focusing more directly on getting this repo updated, issues managed, and PRs merged. Think of us as being 'sponsored' by HDC to support the project (and actually get the updated code into the theme directory).

To reiterate, most of our agency's work is custom WP development. We use UnderStrap as a parent theme and generally start with the default UnderStrap child theme. I think that big picture, I see UnderStrap serving in a similar role to Genesis, in that it's a framework for developers who want to have something reliable and extensible, with a great community and some decent off-the-shelf tools available. On the other hand, I see Genesis as a little too idiosyncratic and the barrier-to-entry a little too high. The thing that Bootstrap, Underscores, and WordPress all have in common is that they are great for new developers who are eager to learn, as well as seasoned pros who plan on commenting out half of the @imports in the sass anyway. I want to honor that mindset. (The other difference here is that Genesis is completely nonfunctional without a child theme, whereas UnderStrap is its own opinionated theme available in the directory. So that's something we're keeping in mind.)

That being said, that's just a general look into our thought process. Please invite me to the Slack and lets keep chatting big picture or specifics. I understand that how WE use UnderStrap is not how EVERYONE uses UnderStrap. Here's our major next steps, though we welcome discussion on any of this. I'll also be blending in a few things that are outside of the realm of this repo, but just putting it all here to help you get an idea of where our head is at.

- [x] Survey the current UnderStrap (paid and free) audience to learn more about them.
- [x] Turn the Roadmap feature back on for parent theme development (this is done, so let's fill it up)
- [ ] Populate a few top-level items in the Roadmap (my top priorities: optional BS5, Gutenberg support, cleaning up the NPM build process)
- [ ] Respond to (and sometimes close) the older issues on here
- [ ] Freshen up the public understrap.com website to attract our next wave of UnderStrap users and clean out anything that isn't directly related to the core project
- [ ] Update the documentation
- [ ] Examine the larger ecosystem and make some decisions with 

Looking forward to the future of this project.

**Comment by howarddc** [comment]
Thanks, @danemorgan and @bacoords!

@Thomas-A-Reinert, just to confirm publicly here, we have a team of 15 that includes 10 employees and 5 contractors, so you'll see many folks from Howard Development & Consulting working on UnderStrap, and Brian, Dane and I will be taking the lead. We're treating UnderStrap just like any other part of the job for everyone on our team, which I think will be really positive for everyone 👍

**Comment by mattneal-stafflink** [comment]
It makes me so happy to see this project back up and running I started using Understrap a few years ago when I was the team lead for our WordPress side of the agency. I recently took a new role (well, a year ago) at a new business with no web team. We're now a team of three and exponentially growing with our next big clients, and I've been struggling with what framework/starter theme to use. Seeing Understrap failing was truly heartbreaking, so all of this is music to my ears.

I'm a PHP/SCSS/JS/WP developer with a heavy focus on UI/UX (my tertiary education was design) and I'd love to contribute in some capacity in my spare time where I can. 

Thanks, @howarddc and your team for taking the steps to breathe life back into this project, I truly appreciate the thought and financial effort you've undertaken here.

Outside of the obvious tasks at hand, I think getting the documentation on getting started should be cleared up a bit where possible. It's has been in an incomplete state for the entirety of this project, I think.

**Comment by tuberb** [comment]
As a developer that relied on Understrap to build sites for several years, and supported it financially in a small way, but had to move on to other theme frameworks once Understrap stopped being stable, I was glad to see the news about the new ownership and revitalized support. 

One question - the new theme version is great, but will you be releasing an update to the child theme on Github, which I used as the basis for all of my clients' custom themes? Once that is up I can spend some time checking out the new versions, and possible come back to Understrap for future projects.

Thanks,

Barry

**Comment by danemorgan** [comment]
Hi @tuberb

We will absolutely be updating the child theme. We are aware that it is essentially broken right now. First, we want to get the dependencies taken care of in the main theme as a number of them are reported as security issues. But after that, work on the child theme and updating to Bootstrap 5 are top concerns.

**Comment by danemorgan** [comment]
Hi @mattneal-stafflink

We agree that the documentation needs to happen. It's difficult to grow if people don't know how to get started and have to try to figure it out blind.

**Comment by Thomas-A-Reinert** [comment]
@danemorgan  - some documentation is already available.
* http://usdocu.tarthemes.com/
* https://github.com/Thomas-A-Reinert/Understrap-Documentation
It was done so far by @axlright , @0dp and me and is probably not fully complete.
